src/guide/components/props.md

```diff
@@ -603,11 +603,11 @@ defineProps({
 ```js
 export default {
   props: {
-    disabled: [Boolean, Number]
+    disabled: [String, Boolean, Number]
   }
 }
 ```
 
 </div>
 
-The casting rules for `Boolean` will apply regardless of type appearance order.
+The casting rules for `Boolean` will apply if it occurs prior to `String`.
```

Comment:
I agree that the wording needs to change, but I think we need to be a little more precise here. The proposed change would imply that `disabled: [Boolean, Number]` would not use the casting rules, because `Boolean` is not prior to `String`. Perhaps something like this?

```suggestion
If the array of types includes `Boolean` then the casting rules for `Boolean` will apply, unless `String` appears earlier in the array.
```